### ISSUE TITLE
Bitcoin Socratic Seminar #32 Ideas

### ISSUE THREAD

**Issue by sbddesign** [atlantabitdevs/atlbit.dev#60]
What should we talk about at Socratic Seminar 32?

**Comment by sbddesign** [comment]
We can have an opening demo of #69 Blockstamp

**Comment by sbddesign** [comment]
[Gloria gets a package relay TX mined on testnet4](https://x.com/glozow/status/1829100551067365608)

**Comment by sbddesign** [comment]
[Bitcoin Mining Development Mailing List Motivations](https://groups.google.com/g/bitcoinminingdev/c/97fkfVmHWYU?pli=1)

**Comment by sbddesign** [comment]
[Strike BOLT 12 Blog Post](https://strike.me/blog/bolt12-offers/)

**Comment by sbddesign** [comment]
[Anduro Whitepaper](https://coordinate.anduro.io/)

**Comment by alexlwn123** [comment]
[Lava Loans Protocol V2](https://github.com/lava-xyz/loans-paper/blob/960b91af83513f6a17d87904457e7a9e786b21e0/loans_v2.pdf)

**Comment by alexlwn123** [comment]
[Benny's DLC Dev Kit is funded](https://x.com/bennyhodl/status/1831720708042260618)

**Comment by alexlwn123** [comment]
[The data availability problem for ZK rollups](https://bitcoinmagazine.com/technical/bitcoin-rollups-the-rock-or-the-hard-place)

tldr: Working with witnesses for offchain computations introduces some DoS scenarios that are new for bitcoin.

**Comment by alexlwn123** [comment]
Wasabi Coordinator Ecosystems
https://wabisator.com/ 
https://wasabist.io/

**Comment by sbddesign** [comment]
And that's a wrap on September
